sfdc/Package.md

```diff
@@ -1,501 +1,22 @@
-Connects to Salesforce from Ballerina.
+Connects to Salesforce from Ballerina
 
-## Module Overview
-The Salesforce connector allows users to perform CRUD operations for SObjects, query using SOQL, search using SOSL, 
-and describe SObjects and organizational data through the Salesforce REST API. Also, it supports adding bulk data jobs 
-and batches of types JSON, XML, and CSV via the Salesforce Bulk API. Apart from these functionalities Ballerina 
-Salesforce Connector includes a listener module to capture events. This connector follows OAuth 2.0 authentication for 
-secure access. 
+## Package overview
+The `ballerinax/sfdc` is a [Ballerina](https://ballerina.io/) connector for Google Drive.
 
-## Compatibility
-|                     |    Version                  |
-|:-------------------:|:---------------------------:|
-| Ballerina Language  | swan-lake-Alpha5            |
-| Salesforce API      | v48.0                       |
-| Salesforce Bulk API | v1                          |
+This package performs CRUD operations for SObjects, query using SOQL, search using SOSL and, describe SObjects, and organizational data through the Salesforce REST API. Also, it supports adding bulk data jobs and soap api operations.
```

Comment:
SOAP API

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/156/commits/e4b1d0a837a4ba241b6f7ece1a4b730aa64222e5